### DISCUSSION TITLE
Clicking away to close the window

### DISCUSSION THREAD

**Discussion by AntoninLaudon** [seerge/g-helper#3253] [Ideas]
Hello, I love G-helper but I was wondering if there was a setting to close the window when I click away.
For example, right clicking opens a popup that dissapears when I click on the desktop and It would be cool to have this with the 
main GHelper window to.
See video below :)

https://github.com/user-attachments/assets/6b1cb90c-6d19-4479-81f2-357c0c9d0634

Thanks!

**Comment by seerge** [comment]
@AntoninLaudon hello,

G-Helper already has same menu that disappears when you click outside. You can call it with right-mouse click on a tray icon.

**Reply by AntoninLaudon** [reply]
Yes but I was wondering if it was possible with the main window that's called when left clicking. Not the smal window that's called when right clicking

**Reply by seerge** [reply]
@AntoninLaudon main window behaves same as any other window in ... windows :) It it has too many UI elements (like dropdowns or track-bars) that won't work correctly if window will close on losing focus. Also G-Helper has secondary windows like `Fans+Power` that also make such behavior very confusing.

**Reply by AntoninLaudon** [reply]
Oh okay, thanks for answering !
Have a nice day and thanks again for that wonderful tool !
